### DISCUSSION TITLE
array null, when it isn't.

### DISCUSSION THREAD

**Discussion by LazyGamer1111** [Minestom/Minestom#1381] [Help] [answered]
When i use my function, it says in console Cannot store to object array because "com.github.LazyGamer111.minestom_java.player.fakePlayer.playerInfoPacketAll" is null

the fakeplayer script it is referring to is https://hastebin.com/ojetigudog.java

What did i do wrong in this situation?

**Comment by GreatWyrm** [comment, marked as answer]
From what I can see in the code, the array is null.

You never instantiated the `playerInfoPacketAll` array with a size.
You will need to change the line to something like: `public static PlayerInfoPacket[] playerInfoPacketAll = new PlayerInfoPacket[10];`

Also, you don't have to write your FakePlayer class, you can use Minestom's in-built class.

**Reply by GoldenStack** [reply]
better answer than mine frfr

**Reply by LazyGamer1111** [reply]
Thanks, pretty new to java.

**Comment by GoldenStack** [comment]
You never initialize the array. It is null.

**Comment by TheMode** [comment]
Please include code snippet directly in the discussion (https://docs.github.com/en/get-started/writing-on-github/working-with-advanced-formatting/creating-and-highlighting-code-blocks) instead of using temporary hosts
